Python/Age Calculator/Age_Calculator_GUI.py

```diff
@@ -0,0 +1,197 @@
+# import all functions from the tkinter
+from tkinter import *
+from tkinter import messagebox
+from tkinter import ttk
+
+# import time date module
+from datetime import date
+
+# Function for clearing the
+# contents of all text entry boxes
+
+
+def clearAll():
+
+    dayField.delete(0, END)
+    monthField.delete(0, END)
+    yearField.delete(0, END)
+    givenDayField.delete(0, END)
+    givenMonthField.delete(0, END)
+    givenYearField.delete(0, END)
+    rsltDayField.delete(0, END)
+    rsltMonthField.delete(0, END)
+    rsltYearField.delete(0, END)
+
+
+# function for checking error
+def checkError():
+
+    if (dayField.get() == "" or monthField.get() == ""
+        or yearField.get() == "" or givenDayField.get() == ""
+            or givenMonthField.get() == "" or givenYearField.get() == ""):
+
+        # error message
+        messagebox.showerror("Input Error")
+
+        # clearAll function calling
+        clearAll()
+
+        return -1
+
+# Function to Calculate the age
+
+
+def calculateAge():
+
+    # check error
+    value = checkError()
+
+    if value == -1:
+        return
+
+    else:
+        # take a value from the respective entry boxes
+        birth_day = int(dayField.get())
+        birth_month = int(monthField.get())
+        birth_year = int(yearField.get())
+
+        given_day = int(givenDayField.get())
+        given_month = int(givenMonthField.get())
+        given_year = int(givenYearField.get())
+
+        # if birth date is greater then given birth_month
+        # then donot count this month and add 30 to the date so
+        # as to subtract the date and get the remaining days
+        month = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
+
+        if (birth_day > given_day):
+            given_month = given_month - 1
+            given_day = given_day + month[birth_month-1]
+
+        # if birth month exceeds given month
+        if (birth_month > given_month):
+            given_year = given_year - 1
+            given_month = given_month + 12
+
+        # calculate day, month, year
+        calculated_day = given_day - birth_day
+        calculated_month = given_month - birth_month
+        calculated_year = given_year - birth_year
+
+        # insert method inserting the
+
+        rsltDayField.insert(10, str(calculated_day))
+        rsltMonthField.insert(10, str(calculated_month))
+        rsltYearField.insert(10, str(calculated_year))
+
+
+# The Driver Code
+if __name__ == '__main__':
+
+    # Creating the GUI window
+    root = Tk()
+
+    # Setting the background color
+    root.config(background='pink')
```

Comment:
I suggest you, not to give any `color` attribute.
It will look good the default one!

Reply:
Sorry, I did not mention anything regarding the background. In fact, you have deleted around 613 files in the repo, so I wanted you to revert that back.

Reply:
I have deleted anything?

Reply:
@mehabhalodiya  do you want me to remove background color?

Reply:
> I have deleted anything? @GowthamGoush

Yeah, in your final commit, around 613 files were deleted, so consider resetting the head to one commit before.